javalanglib/src/main/scala/java/lang/_String.scala

```diff
@@ -734,6 +734,215 @@ for (cp <- 0 to Character.MAX_CODE_POINT) {
     }
   }
 
+  def stripLeading(): String = {
+    length() match {
+      case 0                                                => thisString
+      case 1 if java.lang.Character.isWhitespace(charAt(0)) => ""
+      case n                                                =>
+        var idx = 0
+        while (idx < n && Character.isWhitespace(charAt(idx))) idx += 1
+        substring(idx)
+    }
+  }
+
+  def stripTrailing(): String = {
+    length() match {
+      case 0                                                => thisString
+      case 1 if java.lang.Character.isWhitespace(charAt(0)) => ""
+      case n                                                =>
+        var idx = n - 1
+        while (idx >= 0 && Character.isWhitespace(charAt(idx))) idx -= 1
+        substring(0, idx + 1)
+    }
+  }
+
+  def strip(): String = {
+    length() match {
+      case 0                                                => thisString
+      case 1 if java.lang.Character.isWhitespace(charAt(0)) => ""
+      case n                                                =>
+        var leading = 0
+        while (leading < n && Character.isWhitespace(charAt(leading))) leading += 1
+        if (leading == n) ""
+        else {
+          var trailing = n - 1
+          while (trailing >= 0 && Character.isWhitespace(charAt(trailing))) trailing -= 1
+          if (leading == 0 && trailing + 1 == n) thisString
+          else substring(leading, trailing + 1)
+        }
+    }
+  }
+
+  def isBlank(): scala.Boolean = {
+    val len = length()
+    var start = 0
+    while (start != len && java.lang.Character.isWhitespace(charAt(start))) start += 1
```

Comment:
Done, I'm fully qualifying for `def transform[R](f: java.util.function.Function[String, R]): R` because I saw `java.lang.StringBuilder` being fully qualified as well, do we want to keep the prefix for that one?